openedx/core/lib/xblock_serializer/block_serializer.py

```diff
@@ -89,6 +89,11 @@ def _serialize_normal_block(self, block) -> etree.Element:
                     with filesystem.open(file_path, 'rb') as fh:
                         data = fh.read()
                     self.static_files.append(StaticFile(name=unit_file.name, data=data, url=None))
+
+        # Serialize and add tag data if any
+        if isinstance(block, TaggedBlockMixin):
+            block.add_tags_to_node(olx_node)
+
```

Comment:
This was missing, and so normal blocks weren't getting their tags serialized, [only HTML blocks were](https://github.com/open-craft/edx-platform/pull/640/files#diff-7d52ae1f43aa6fcf1ca07b918cfe9707b157601c3ec872f3866c65dece4fae6cL118-L120).